spec/acceptance/basic_spec.rb

```diff
@@ -54,4 +54,8 @@ class { 'redis::globals':
     its(:exit_status) { is_expected.to eq 0 }
   end
 
+ describe command("DJANGO_SETTINGS_MODULE=pulpcore.app.settings PULP_SETTINGS=/etc/pulp/settings.py python3-django-admin dumpdata auth.User | grep 'auth.user'") do
```

Comment:
You can avoid using grep here:

```suggestion
 describe command("DJANGO_SETTINGS_MODULE=pulpcore.app.settings PULP_SETTINGS=/etc/pulp/settings.py python3-django-admin dumpdata auth.User") do
   its(:stdout) { is_expected.to match(/auth\.user/) }
```

This is useful when it fails. Grep doesn't show non-matches and it'll just be a silent failure. Using the matcher it'll show a usable error.

Reply:
I agree this approach is better... thanks for pointing it out.

Reply:
Updated.. :+1:

Reply:
Thanks @sjha4 and @ekohl ! Will merge once tests are complete.